app/src/models/branch.ts

```diff
@@ -41,31 +41,21 @@ export function eligibleForFastForward(
   )
 }
 
-/** A branch as loaded from Git. */
+/**
+ * A branch as loaded from Git.
+ *
+ * @param name The short name of the branch. E.g., `master`.
```

Comment:
I _think_ this should be documenting the `constructor` element below, not the other `class` itself.

Reply:
@Daniel-McCarthy friendly bump on this change - it'd be nice to close this out before we open more PRs like this

Reply:
Ah, of course. Nice catch! Thanks for the feedback and reminding me of this.